_posts/2023-03-27-Gen4-modems-release.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+title: Subnero Launches the Fourth Generation Wireless Networked Communication (WNC) Series of Underwater Acoustic Modems
+banner : images/banner-pulse-standard.jpg
+date:  2023-03-27
+categories: wnc
+thumbnail: images/pulse-thumbnail-gen4.png
+---
+
+Singapore-based company Subnero has launched its [fourth-generation Wireless Networked Communication (WNC) series of underwater acoustic modems](https://subnero.com/products/wnc){:target="_blank"}. 
+
+The upgraded devices provide best-in-class communication performance, advanced networking and localization capabilities, and developer tools to simplify the development and deployment of underwater wireless networks.
+This launch is the culmination of our continuous effort to improve and innovate. We have listened closely to customer feedback and learned valuable insights that have been incorporated into the fourth generation of the WNC product series.
+
+
+The M25M series offers up to 15* kbps and more than 5 km of communication range, while the upcoming S60H series provides up to 30* kbps and over 1 km of range in challenging conditions. These devices feature the latest iteration of UnetStack - UnetStack4 to bring the latest generation of Software-Defined Open Architecture Modems (SDOAM) to the market. The devices provide ease of integration with unified interfaces, APIs in multiple programming languages (Java, Groovy, Julia, Python, C JavaScript), OS/hardware platform independence, and support for external sensors, such as GPS and AHRS, as well as machine learning and artificial intelligence (ML/AI) readiness for edge computing applications.
+
+
+The advanced networking and localization capabilities include operating multiple communication nodes simultaneously, support for multi-hop links, routing protocols, mesh networks, cellular-like networks for broader coverage, and remote control of devices out-of-the-box.
+
+
+Subnero's WNC series of underwater acoustic modems are ideal for various applications, including underwater sensors, marine robotics, diver communication networks, and underwater wireless networks. The Subnero Wireless Integrated Suite (SWIS) - ADCP Edition, provides a seamless end-to-end solution for wireless data transfer from Acoustic Doppler Current Profilers (ADCPs) to avoid uncertainties during long-term deployments. Ripple is a software package that provides a single software interface to all communication devices, both above and underwater on your underwater vehicle, be it wireless or wired.
```

Comment:
```suggestion
Subnero's WNC series of underwater acoustic modems are ideal for various applications, including underwater sensors, marine robotics, diver communication networks, and underwater wireless networks. The [Subnero Wireless Integrated Suite (SWIS) - ADCP Edition](https://subnero.com/solutions/swis){:target="_blank"}, provides a seamless end-to-end solution for wireless data transfer from Acoustic Doppler Current Profilers (ADCPs) to avoid uncertainties during long-term deployments. Ripple is a software package that provides a single software interface to all communication devices, both above and underwater on your underwater vehicle, be it wireless or wired.
```